### ISSUE TITLE
Set hood and shooter speed command

### ISSUE THREAD

**Issue by Jacob4649** [FRC4946/2020Robot#82]
We need a command that sets the hood angle setpoint and the shooter speed setpoint, but does not enable the PID on, or require either subsystem

**Comment by matthew-reynolds** [comment]
I imagine that this is for the manual mode preset buttons on the operator stick, correct?

**Comment by matthew-reynolds** [comment]
Isn't the intent of those buttons to:

1. Set **and spin up** the shooter to a preset speed, and
2. Display the desired hood angle on the SmartDashboard

I think that this command should require the shooter subsystem, and should enable the PID on that subsystem. I agree with not enabling PID or requiring the hood though - That will be controller open-loop by the operator stick.

**Comment by Jacob4649** [comment]
Yeah sorry I was thinking we would spin up with the existing spin up button on the operator stick, but that seems unnecessarily complicated, this is definitely better
